pandas/io/json.py

```diff
@@ -123,32 +123,38 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
         file. For file URLs, a host is expected. For instance, a local file
         could be ``file://localhost/path/to/table.json``
 
-    orient
+    orient : string,
+        Indication of expected JSON string format.
+        Compatible JSON strings can be produced by ``to_json()`` with a
+        corresponding orient value.
+        The set of possible orients is:
 
-        * `Series`
+          - ``'split'`` : dict like
```

Comment:
There is some indentation here compared to the base level of "The set of ..." (which is not needed)